### PR TITLE
Fixes Issue#15 no start button in nightly

### LINKED ISSUE #15: Firequery icon not visible in nightly  41.0a1

### PR DESCRIPTION
**Pull request by aliuk2012**
Fixes #15  No start button appeared in nightly build so you couldn't enable the plugin

### CODE REVIEW

File: chrome/locale/en-US/toolbox.properties [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# LOCALIZATION NOTE (fireQuery.title): FireQuery extension title
+firequery.title=FireQuery
+
+# LOCALIZATION NOTE (firequery.button.title, firequery.button.tip):
+# Label and a tooltip for FireQuery start button available in the
+# Firefox toolbar and Style Editor panel. 
+firequery.startButton.title=FireQuery
+firequery.startButton.tip=Remote debugging protocol inspector.
```
Comment (janodvarko) [inline chrome/locale/en-US/toolbox.properties:8]: Change the tooltip to refer to FireQuery

Honza

File: lib/start-button.js [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+      let context = ToolboxChrome.getContext(toolbox);
+      let overlayId = ToolboxOverlay.prototype.overlayId;
+      let overlay = context.getOverlay(overlayId);
+      if (!overlay) {
+        TraceError.sysout("startButton.onToggleFireQuery; ERROR " +
+          " No overlay: " + overlayId, context);
+        return;
+      }
+
+      let win = overlay.toggleInspector();
```
Comment (janodvarko) [inline lib/start-button.js:151]: Another left over from RDPi?

Btw. what the onToggleFireQuery() method is supposed to do?

(E.g. the RDPi opens the console, but FireQuery doesn't have any extra UI)

Honza

**Comment by janodvarko** [comment]
@aliuk2012: made some comments. Also the current icon is very similar to what FirePHP is using
https://addons.mozilla.org/cs/firefox/addon/firephp/

Shouldn't we use at least the one I am seeing on AMO?
https://addons.mozilla.org/cs/firefox/addon/firequery/

Honza

**Comment by aliuk2012** [comment]
Very interesting about the icon looking similar. The icon that is currently in use is the old png one. I'll make the changes and another pull request to bring in the new icons i did last week.

Thanks for comments above I've get those changes made, Actually not sure why this version had a start button (other than maybe if firebug isn't available) but the original one I dont recall having its own start button.

**Comment by janodvarko** [comment]
> Actually not sure why this version had a start button (other than maybe if firebug isn't available)
> but the original one I dont recall having its own start button. 

So, why we are introducing the icon? :-)
Do we need it?

I don't feel like the links to newsgroups are important enough to have that icon ...

Honza

**Comment by aliuk2012** [comment]
Not required anymore as per issue #15

**Pull request closed without merging**